pkg/3scale/amp/operator/redis_options_provider.go

```diff
@@ -51,13 +58,127 @@ func (r *RedisOptionsProvider) GetRedisOptions() (*component.RedisOptions, error
 
 	r.setPersistentVolumeClaimOptions()
 
-	err := r.options.Validate()
+	// Should the operator be reading redis secrets?
+	// When HA is disabled, do we support external redis?
```

Comment:
We do not support external redis when HA (external mode) is disabled.
Users could precreate a secret in that case with urls pointing to external DBs but it's not a supported scenario. Should we document this edge-case? (until now we are assuming the user is responsible enough to not do that when internal dbs are deployed)

Reply:
My reasoning also applies for system database.